middleware/locale/locale_test.go

```diff
@@ -0,0 +1,55 @@
+package locale_test
```

Comment:
Any reason why you're using external package approach ? If not, that should be `package locale`.